src/components/structures/MatrixChat.js

```diff
@@ -607,6 +607,7 @@ module.exports = React.createClass({
     // @param {boolean=} roomInfo.show_settings Makes RoomView show the room settings dialog.
     // @param {string=} roomInfo.event_id ID of the event in this room to show: this will cause a switch to the
     //                                    context of that particular event.
+    // @param {boolean=} roomInfo.highlighted If true, add event_id to the hash of the URL
```

Comment:
surely it means way more than that? like, the event will also be highlighted in the view. 

Bear in mind this is documenting the parameters on the view_room more than the parameters to the function - the  docs just got left behind when view_room started going to more places :/